src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -21,6 +23,7 @@ type Props = {
     };
     dataLinkNamePrefix: string;
     inHeader: boolean;
+    enableAusMoment2020Header: boolean,
```

Comment:
Just noticed this ends in a comma not a semicolon like the others. I’d’ve expected it to be a comma tbh.